srcpkgs/ffmpeg/template

```diff
@@ -224,7 +224,7 @@ ffmpeg-devel_package() {
 	short_desc="Decoding, encoding and streaming software - development files"
 	conflicts="ffmpeg6-devel"
 	replaces="ffmpeg6-devel>=0"
-	if [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
+	if [[ "$XBPS_TARGET_MACHINE" = "i686"* ]]; then
```

Comment:
Our style preference is a case block instead of the Bash test operator. See, for example, the top of the template.

Reply:
I just went with what was in the template. I'll change it.